client/a8c-for-agencies/sections/referrals/lib/get-estimated-commission.ts

```diff
@@ -1,20 +1,8 @@
 import { APIProductFamilyProduct } from '../../../../state/partner-portal/types';
 import { Referral } from '../types';
+import { getProductCommissionPercentage } from './commissions';
 import { getNextPayoutDateActivityWindow } from './get-next-payout-date';
 
-export const getProductCommissionPercentage = ( slug?: string ) => {
```

Comment:
Is this function no longer needed? Seems like we're using `getEstimatedCommission()` instead but just double checking.

Reply:
Oh, I just removed it as it was a duplicate from `./commissions` file, so I imported it instead. We probably would eventually reorganize the code here a bit.